### DISCUSSION TITLE
I need help with applying to Github Student Developer

### DISCUSSION THREAD

**Discussion by xplrds** [community/community#29326] [GitHub Education]
I have trouble uploading a picture to get my application approved every time I try to submit the picture I took it says to upload an dated school ID option. But I wanted to apply using my admissions message that says I got into SJSU but then I keep getting said that I need to upload a dated school ID. My SJSU ID doesn't say a dated on the bottom or anywhere on the card but this is the ID we use to access the campus. If anyone know what I can do to solve this problem please let me know.

Thank you

**Comment by melvincwng** [comment]
@xplrds, my suggestion would be to raise a ticket directly to the GitHub Education Support team over [here](https://support.github.com/contact/education).

Provide a few working days at least for the support engineers to investigate and resolve your issue.

Hope this helps 😊

**Reply by xplrds** [reply]
Thank you Melvin for the suggestion I will do this and see what they tell me.

**Reply by melvincwng** [reply]
@xplrds , no problem, glad that I could help!
